docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -7,21 +7,21 @@ The main media controller features include:
 
 -   Updating and retrieving information
 
-    You can [update the metadata and playback information](#get_media) on the server side, and then retrieve that information on the client side.
+    You can [update the metadata and playback information](#updating-and-retrieving-information) on the server side, and then retrieve that information on the client side.
 
     The media control server provides current information about the registered application that you can send to the client.
 
     When the client requests the information, the media control server updates the state information of an active application before transferring the data. If the application is not running when the client request arrives, the media control server transfers the latest information.
 
 - Sending and processing commands
 
-    You can [send a command](#send_media) to the server from the client side, and then process the command on the server side.
+    You can [send a command](#sending-and-processing-commands) to the server from the client side, and then process the command on the server side.
 
-    The client can request server state or [metadata](#servermetadata) information from the server, and receive it through an event handler.
+    The client can request server state or [metadata](#media-control-metadata-properties) information from the server, and receive it through an event handler.
 
```

Comment:
@hsgwon
In the previous comment I suggested simple description of newly added section to be added here. For example, how about adding like below?
Below is just an example, and sentences are not trimmed yet.
- Updating and Retrieving Playlist
 You can "_create a playlist_" (link to "Updating and Retrieving Playlist" section ) on the server side, and then retrieve that information on the client side.
 The server "_set the playback capability_"(link to "Setting and Getting Playback Capability") and allow the client to get it.

If there are more information about it, you can add it to here.

Reply:
I misunderstood your comment. I'll add it.